### ISSUE TITLE
replace magrittr with native pipes

### ISSUE THREAD

**Issue by fabian-s** [tidyfun/tidyfun#110]

**Comment by jeff-goldsmith** [comment]
makes sense to remove magrittr as a dependency (although i still kinda shudder at native pipes ...)

what's a good timeline for this? will we wait until the new R version is pretty prevalent among users, since it'll be required?

**Comment by fabian-s** [comment]
> makes sense to remove magrittr as a dependency (although i still kinda shudder at native pipes ...)

yeah, but not sure it makes sense anymore  because native pipe can't do stuff like `x  %>% { some_function(bla, bloo, ble, .x) }` which I think I've used a number of times, and the magrittr-pipe comes with the other tidyverse-imports anyway, I think....

> what's a good timeline for this? will we wait until the new R version is pretty prevalent among users, since it'll be required?
definitely wait a while

**Comment by fabian-s** [comment]
closing this  for now. might have to revisit in context of  #69
